### PR TITLE
Feature/chat

### PR DESCRIPTION
**Pull request by Middi**
created database search by user and room ID

### CODE REVIEW

File: client/src/app/actions/index.js [hunk 1]
```diff
@@ -12,9 +12,16 @@ export const getCurriculum = () => async dispatch => {
 };
 
 // Messages Actions
-export const getConversations = () => async dispatch => {
+export const getConversations = (id) => async dispatch => {
```
Comment (chazmcgrill) [inline client/src/app/actions/index.js]: This should be user i think?

File: client/src/app/pages/Inbox.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import { connect } from 'react-redux';
+import { getConversations } from '../actions';
+import requireAuth from './requireAuth';
+
+class Conversations extends Component {
+  componentDidMount() {
+    this.props.fetchConversations(this.props.user);
+  }
+
+  roomClick = (e) => {
```
Comment (chazmcgrill) [inline client/src/app/pages/Inbox.js]: Nice, make the magic e parameter into roomId for readability

File: client/src/app/actions/index.js (hunk 1 above)
Comment (Middi) [reply client/src/app/actions/index.js]: fixed on new push

File: client/src/app/pages/Inbox.js (hunk 2 above)
Comment (Middi) [reply client/src/app/pages/Inbox.js]: fixed on new push

File: client/src/app/pages/Inbox.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+    return (
+      <main>
+        
+        <h1>Inbox</h1>
+
+         {conversations.map(convo => (
+             <Fragment
+             key={convo._id}>
+            <h3 onClick={e => this.roomClick(convo.roomId)}>
```
[the remark this answers is not in the document]
Comment (Middi) [reply client/src/app/pages/Inbox.js]: all fixed

**Pull request merged**